docs/sphinx/user_guide/cook_book/reduction.rst

```diff
@@ -0,0 +1,93 @@
+.. ##
+.. ## Copyright (c) 2016-24, Lawrence Livermore National Security, LLC
+.. ## and other RAJA project contributors. See the RAJA/LICENSE file
+.. ## for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+.. ##
+
+.. _cook-book-reductions-label:
+
+=======================
+Cooking with Reductions
+=======================
+
+Please see the following section for more info on RAJA reductions:
+
+ * :ref:`feat-reductions-label`.
+
+
+----------------------------
+Reductions with RAJA::forall
+----------------------------
+
```

Comment:
Is the plan to expand this over time to show how to use the concretizers and other features?  Also, it would be helpful to users to provide guidance on choosing exec policies for reduction cases.

Reply:
Ya the intention is to add more sections over time and this page should have guidance on choosing exec polices for use with reductions.